package.json

```diff
@@ -3,79 +3,69 @@
   "version": "0.0.0",
   "description": "Eplant",
   "main": "index.ts",
+  "type": "module",
   "scripts": {
     "test": "jest --passWithNoTests",
-    "storybook": "start-storybook -p 6006",
-    "start": "vite serve",
-    "build": "tsc --noEmit && vite build",
-    "build-storybook": "build-storybook",
-    "postinstall": "cd ./node_modules/flexlayout-react && npm install --legacy-peer-deps && npm run build:commonjs"
+    "dev": "vite",
+    "build": "tsc && vite build",
```

Comment:
Was [`--noEmit`](https://www.typescriptlang.org/tsconfig#noEmit) removed on purpose? If not, should we add it back into the build command?

> --noEmit: Do not emit compiler output files like JavaScript source code, source-maps or declarations.  This makes room for another tool like [Babel](https://babeljs.io/), or [swc](https://github.com/swc-project/swc) to handle converting the TypeScript file to a file which can run inside a JavaScript environment. You can then use TypeScript as a tool for providing editor integration, and as a source code type-checker.